src/sql/parts/grid/views/editData/editData.component.ts

```diff
@@ -228,7 +236,7 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 
 		let cellSelectTasks: Promise<void> = Promise.resolve();
 
-		if (this.currentCell.isEditable && this.currentEditCellValue !== null && !this.removingNewRow) {
+		if (this.currentCell.isEditable && this.currentEditCellValue !== null && !this.removingNewRow && !this.revertingRow) {
```

Comment:
why is `this.revertingRow` necessary? We won't enter this if statement anyway because the new code sets `this.currentEditCellValue` to null as soon as you hit escape.

Is it because this method is getting called when we call `self.focusCell` in `onRevertRow()`? In that case we could just set `this.currentEditCellValue` to null before calling `focusCell`